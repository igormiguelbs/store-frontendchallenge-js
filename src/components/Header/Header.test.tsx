import renderer from "react-test-renderer";
import { Header } from ".";

describe("/Header", () => {
  it("should render component correctly", () => {
    const result = renderer.create(<Header />);
    expect(result).toMatchSnapshot();
  });
});
