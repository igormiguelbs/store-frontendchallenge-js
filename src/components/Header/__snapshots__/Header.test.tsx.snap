// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`/Header should render component correctly 1`] = `
<div
  className="sc-bcXHqe fPcNxG"
>
  <h1>
    MKS 
    <span>
      Sistemas
    </span>
  </h1>
</div>
`;
